Code/autopkglib/AppDmgVersioner.py

```diff
@@ -15,25 +15,22 @@
 # limitations under the License.
 """See docstring for AppDmgVersioner class"""
 
-from __future__ import print_function
-
 import glob
 import os.path
 
-from autopkglib import ProcessorError
+from autopkglib import ProcessorError, log
 from autopkglib.DmgMounter import DmgMounter
 
-
 # pylint: disable=no-name-in-module
 try:
     from Foundation import NSData, NSPropertyListSerialization
     from Foundation import NSPropertyListMutableContainers
 except:
-    print(
+    log(
```

Comment:
All processors should use self.output() instead of log/log_err.

Reply:
On the surface that sounds correct/reasonable. There _might_ be reasons I have forgotten where a processor needs to bypass self.output(), however. So I'm hesitant to say: "sure, go ahead and do a mass find-and-replace". 

Let's not get so enamored of our consistency that we introduce regressions because we quickly "conform" all the code.

Reply:
That sounds like a reasonable response to me. We can always come by later and change individual processors if there's a feeling that they're too "noisy" or not respectful enough of AutoPkg verbosity settings.